src/Admin/BlockAdmin.php

```diff
@@ -32,6 +32,8 @@
  * Admin class for the Block model.
  *
  * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * @final
```

Comment:
```suggestion
 * @final since sonata-project/page-bundle 3.x
```

Same apply for the rest of final keywords.